src/server/game/Movement/MovementGenerators/TargetedMovementGenerator.cpp

```diff
@@ -73,8 +73,8 @@ void TargetedMovementGeneratorMedium<T, D>::_setTargetLocation(T* owner, bool up
             //   doing a "dance" while fighting
             if (owner->IsPet() && i_target->GetTypeId() == TYPEID_PLAYER)
             {
-                dist = i_target->GetCombatReach();
-                size = i_target->GetCombatReach() - i_target->GetObjectSize();
+                dist = 1.0f; //i_target->GetCombatReach();
```

Comment:
"please don't add comments that don't give any additional info then" was about a comment that the original author removed, not to your code. now it all look weird o.O